LoRaEngine/test/LoRaWan.IntegrationTest/TestLogger.cs

```diff
@@ -17,15 +17,13 @@ static class TestLogger
         /// <param name="text"></param>
         internal static void Log(string text)
         {
-            // Log to diagnostics if a debbuger is attached
-            if (Debugger.IsAttached)
+            Console.WriteLine(text);
+
```

Comment:
Any reason why you reverted this change?

Reply:
No, I feel it might be due to the multiple rebase i had to do, should I revert it to the old version?

Reply:
My version looks like this:
```
// Log to diagnostics if a debbuger is attached
            if (Debugger.IsAttached)
            {
                System.Diagnostics.Debug.WriteLine(text);
            }
            else
            {
                Console.WriteLine(text);
            }
```

Up to you. I will fix with my CI PR later this week.